chat-client/src/client/mynahUi.ts

```diff
@@ -131,6 +143,60 @@ export const createMynahUi = (messager: Messager, tabFactory: TabFactory): [Myna
             }
             messager.onVote(payload)
         },
+        onSendFeedback: (tabId, feedbackPayload, eventId) => {
+            if (feedbackPayload.comment !== undefined) {
+                const payload: FeedbackParams = {
+                    tabId,
+                    feedbackPayload,
+                    eventId,
+                }
+                messager.onSendFeedback(payload)
+
+                mynahUi.notify({
+                    type: NotificationType.INFO,
+                    title: 'Your feedback is sent',
+                    content: 'Thanks for your feedback.',
+                })
+            }
+        },
+        onLinkClick: (tabId, messageId, link, mouseEvent, eventId) => {
+            mouseEvent?.preventDefault()
```

Comment:
What is the behaviour for link clicks when we prevent default and stop propagation?

Reply:
I could see the chat-client having additional support for this, but I think it goes back to how destinations wants to handle them.

I'm not entirely sure if this is even needed, but it seems to be useful at least for the VSCode implementation